pytypes/typechecker.pyi

```diff
@@ -0,0 +1,48 @@
+# Generated using mypy stubgen, then editing types for typechecked and
+# no_type_check.
+
+import __builtin__ as builtins
+import types
+import typing
+from . import type_util as type_util, util as util
+from .exceptions import InputTypeError as InputTypeError, OverrideError as OverrideError, ReturnTypeError as ReturnTypeError
+from .type_util import TypeAgent as TypeAgent, deep_type as deep_type, get_orig_class as get_orig_class, has_type_hints as has_type_hints, is_Tuple as is_Tuple, is_builtin_type as is_builtin_type, type_str as type_str
+from .util import getargspecs as getargspecs
+from typing import Any, Optional, Callable, TypeVar, Union, overload
+from warnings import warn as warn
+
+_python___import__ = builtins.__import__
+
+def _pytypes___import__(name: Any, globls: Optional[Any] = ..., locls: Optional[Any] = ..., fromlist: Any = ..., level: int = ...): ...
+
+class _DelayedCheck:
+    func: Any = ...
+    method: Any = ...
+    class_name: Any = ...
+    base_method: Any = ...
+    base_class: Any = ...
+    exc_info: Any = ...
+    raising_module_name: Any = ...
+    def __init__(self, func: Any, method: Any, class_name: Any, base_method: Any, base_class: Any, exc_info: Any) -> None: ...
+    def run_check(self, raise_NameError: bool = ...) -> None: ...
+
+def override(func: Any, auto: bool = ...): ...
+def typechecked_func(func: Any, force: bool = ..., argType: Optional[Any] = ..., resType: Optional[Any] = ..., prop_getter: bool = ...): ...
+def typechecked_class(cls, force: bool = ..., force_recursive: bool = ...): ...
+def typechecked_module(md: Any, force_recursive: bool = ...): ...
+
+T = TypeVar('T', Callable, property, type, types.ModuleType)
+@overload
+def typechecked(memb: str) -> Union[str, types.ModuleType]: ...
+@overload
```

Comment:
The overload should be eliminated according to #97 .

Reply:
:+1: I sent a PR for that one and can return to this after those changes are merged.

Reply:
Done. https://github.com/Stewori/pytypes/commit/40049258d5b58062b7d06efe3138093bf2a25b31